### PR TITLE
0.9.0

### PR DESCRIPTION
**Pull request by BenFradet**
- [x] rename pkg
- [x] prepared for release
- [x] blog post
- [x] wiki
- [x] push wiki
- [x] integration tests
  - [x] load test
  - [x] new region
  - [x] chinese region
  - [x] signing
  - [x] ssl
  - [x] #25 
  - [x] tcp 5.x
  - [x] tcp 2.x

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Added new EnrichedEvent fields (snowplow/snowplow#1345)
+Stopped verifying number of fields in enriched event (snowplow/snowplow#1333)
+Changed organization to com.snowplowanalytics in BuildSettings (snowplow/snowplow#1279)
+Renamed application.conf.example to config.hocon.sample (snowplow/snowplow#1244)
+Added License-2.0.txt (snowplow/snowplow#1329)
+
+Version 0.1.0 (2014-11-26)
+--------------------------
+Fixed order of latitude and longitude
+Created Kinesis Elasticsearch Sink (snowplow/snowplow#1120, snowplow/snowplow#1121)
```
Comment (alexanderdean) [inline CHANGELOG]: Nice

**Comment by BenFradet** [comment]
@jbeemster @chuwy could you have a look, thanks :+1:

**Comment by BenFradet** [comment]
This is ready to go, only pieces left are code and blog post reviews :+1:

**Comment by alexanderdean** [comment]
Great work, @chuwy is up for the code review... I will take a look at the blog post, hopefully tomorrow.

File: http/src/test/scala/com.snowplowanalytics.snowplow.storage.kinesis.elasticsearch/clients/ElasticsearchSenderHTTPSpec.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/**
```
Comment (BenFradet) [inline http/src/test/scala/com.snowplowanalytics.snowplow.storage.kinesis.elasticsearch/clients/ElasticsearchSenderHTTPSpec.scala]: mmh this is passing locally and on travis

Comment (chuwy) [inline http/src/test/scala/com.snowplowanalytics.snowplow.storage.kinesis.elasticsearch/clients/ElasticsearchSenderHTTPSpec.scala]: Maybe tags and branch have different work trees and you pushed only tags recently?

File: core/src/main/scala/com.snowplowanalytics.elasticsearch/loader/BadRow.scala [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ */
+
+package com.snowplowanalytics.elasticsearch.loader
+
+// Scalaz
+import scalaz._
+import Scalaz._
+
+// json4s
+import org.json4s._
```
Comment (chuwy) [inline core/src/main/scala/com.snowplowanalytics.elasticsearch/loader/BadRow.scala]: There's quite few unused imports, btw. You can use `-Ywarn-unused-import` to identify and clean them.

File: core/src/main/scala/com.snowplowanalytics.elasticsearch/loader/clients/Elastic4sSender.scala [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+import scala.util.{Failure, Success}
+
+// Scalaz
+import scalaz._
+import Scalaz._
+import scalaz.concurrent.{Strategy, Task}
+
+// Amz
```
Comment (BenFradet) [inline core/src/main/scala/com.snowplowanalytics.elasticsearch/loader/clients/Elastic4sSender.scala]: yup, me too

**Comment by BenFradet** [comment]
@chuwy I think I've taken care of everything except your failing spec, I wasn't able to reproduce even on a clean clone

File: build.sbt [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+lazy val buildSettings = Seq(
+  organization  := "com.snowplowanalytics",
+  name          := "snowplow-elasticsearch-loader",
+  version       := "0.9.0-rc4",
+  description   := "Load the contents of a Kinesis stream to Elasticsearch",
+  scalaVersion  := "2.11.11",
+  scalacOptions := BuildSettings.compilerOptions,
+  javacOptions  := BuildSettings.javaCompilerOptions,
+  resolvers     += Resolver.jcenterRepo,
+  shellPrompt   := { _ => "elasticsearch-loader> " }
```
Comment (chuwy) [inline build.sbt]: I still think that  `moduleName.value + "> "` is better than `"elasticsearch-loader> "` as you have an idea on what SBT subproject you're working on.

**Comment by chuwy** [comment]
@BenFradet looks great! Have no idea what it was with specs, it works on clean project now.

File: build.sbt (hunk 5 above)
Comment (BenFradet) [inline build.sbt]: problem is that this becomes `snowplow-elasticsearch-loader{core,http,tcp}` which is a bit long imo.

I haven't figured out a way to get to the project id (root, core, etc).

**Comment by alexanderdean** [comment]
What needs to happen to get this released?

**Comment by BenFradet** [comment]
Correct me if I'm wrong @chuwy but this is good to go, I'll release it later today.

**Comment by chuwy** [comment]
Good to go!

**Pull request merged**